### PR TITLE
fix(issue-details): Small page fixes from existing issue details

### PR DESCRIPTION
**Pull request by leeandher**
- Fixed height for all frames for more uniformity

image

- Missing stack trace no longer displays as an error, or says 'unknown'

image

 - Avoid displaying `+Infinity%` since that is technically right, but kinda useless
 
image

- Add an icon to indicate when a group has been shared

image

### CODE REVIEW

File: static/app/components/events/interfaces/frame/deprecatedLine.tsx [hunk 1]
```diff
@@ -524,7 +524,7 @@ const DefaultLine = styled('div')<{
   justify-content: space-between;
   align-items: center;
   background: ${p => (p.isSubFrame ? `${p.theme.surface100}` : `${p.theme.surface200}`)};
-  min-height: 32px;
```
Comment (scttcper) [inline static/app/components/events/interfaces/frame/deprecatedLine.tsx:527]: image

File: static/app/views/issueDetails/actions/shareModal.tsx [hunk 2]
```diff
@@ -28,6 +28,12 @@ interface ShareIssueModalProps extends ModalRenderProps {
 
 type UrlRef = React.ElementRef<typeof AutoSelectText>;
 
+export function getShareUrl(group: Group) {
+  const path = `/share/issue/${group!.shareId}/`;
```
Comment (scttcper) [inline static/app/views/issueDetails/actions/shareModal.tsx:32]:
```suggestion
  const path = `/share/issue/${group.shareId}/`;
```

File: static/app/components/events/interfaces/frame/deprecatedLine.tsx (hunk 1 above)
Comment (leeandher) [reply static/app/components/events/interfaces/frame/deprecatedLine.tsx:527]: nevermind doesn't seem like this can be done consistently.

Comment (leeandher) [reply static/app/components/events/interfaces/frame/deprecatedLine.tsx:527]: Changed to `size='zero'` with some font size changes

**Review by scttcper — APPROVED** [review]

**Pull request merged**